landa/water_body_management/report/catch_log_days_and_counts/catch_log_days_and_counts.py

```diff
@@ -37,50 +38,174 @@
 		"label": "Fishing Area",
 		"options": "Fishing Area",
 	},
-	{
-		"fieldname": "organization",
-		"fieldtype": "Data",
-		"label": "Organization",
-		"options": "Organization",
-	},
-	{
-		"fieldname": "origin_of_catch_log_entry",
-		"fieldtype": "Data",
-		"label": "Origin of Catch Log Entry",
-	},
 	{
 		"fieldname": "fishing_days",
 		"fieldtype": "Int",
 		"label": "Fishing Days",
 	},
+	{  # displayed only for regional or state employees
+		"fieldname": "by_foreign_regional_org",
+		"fieldtype": "Percent",
+		"label": _("Share of other Regional Organizations"),
```

Comment:
```suggestion
		"label": "Share of other Regional Organizations",
```

Daher kommt der Semgrep-Fehler. In einer globalen Konstante darfst du die Übersetzungsfunktion nicht aufrufen. (Weil das Ergebnis der Übersetzungsfunktion vom aktuellen Nutzer abhängen sollte, statt global konstant zu sein)

Reply:
![grafik](https://github.com/alyf-de/landa/assets/95628567/21d328b2-5a5f-445f-881b-37b36584c534)

Deswegen hab ich das hinzugefügt. Du meintest ja, ich soll die fehlgeschlagenen Tests beheben.